bin/snapshot-backups/site-snapshot-download

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 # we might be running in circleci (really?)
```

Comment:
You can take out the "(really?)" here -- that was me. Apparently we really might be.